docs/download.md

```diff
@@ -21,13 +21,13 @@ dev/beta releases of Pulsar or [build from source](https://pulsar-edit.dev/docs/
 ## Cirrus CI binaries
 
 Binaries are produced by [Cirrus CI](https://cirrus-ci.com/github/pulsar-edit/pulsar)
-for each commit to the Pulsar repository.  
+for each commit to the Pulsar repository.\
```

Comment:
```suggestion
for each commit to the Pulsar repository.
```

Reply:
oh I did this intentionally, initially it had 2 trailing spaces which is a newline in markdown, but my editor (and probably many others) trims trailing whitespace, so I replaced it with a backslash in my second commit, which is also a newline in markdown

Reply:
That seems a bit bad if an editor with markdown support is trimming it. If it is then probably worth adding it back in GH simply because we want to have it consistent.

Reply:
I mean if the issue is including a newline, why not put two carriage returns? That'd give us a newline without breaking existing style

Reply:
I think two newlines is like...

this knd of break, versus
this kind of break, if that makes sense?

The intent was to put one line break, but markdown ignores one linebreak only because it can be used to not have like, infinitely massive lines in (programming) text editors? so two spaces past the end or a `\` is used for one line break.

> That seems a bit bad if an editor with markdown support is trimming it. If it is then probably worth adding it back in GH simply because we want to have it consistent.

True, I could language specific override that (or save without formatting). But I still generally think `\` is better, since its visible and not two invisible characters at the end of a line, can be subtle and hard to miss if you're not paying attention